plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/params/ListContainersParams.java

```diff
@@ -0,0 +1,81 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.plugin.docker.client.params;
+
+import org.eclipse.che.plugin.docker.client.json.Filters;
+
+/**
+ * Arguments holder for {@link org.eclipse.che.plugin.docker.client.DockerConnector#listContainers(ListContainersParams)}.
+ *
+ * @author Alexander Andrienko
+ */
+public class ListContainersParams {
```

Comment:
As far as I understand you've tried to align this PR with #952. So it is better to add private constructor and factory method `from()` to this class to avoid different API styles for different docker methods.